Mathlib/FieldTheory/Finite/Basic.lean

```diff
@@ -204,13 +204,10 @@ theorem sum_subgroup_pow_eq_zero [CommRing K] [NoZeroDivisors K]
                   * (Multiset.map (fun i : G => (i.val : K) ^ k) Finset.univ.val).sum = 0 := by
     rw [sub_mul, mul_comm, ← h_multiset_map_sum, one_mul, sub_self]
   rw [mul_eq_zero] at hzero
-  rcases hzero with h | h
-  · contrapose! ha
-    ext
-    rw [← sub_eq_zero]
-    simp_rw [SubmonoidClass.coe_pow, Units.val_pow_eq_pow_val, OneMemClass.coe_one,
-      Units.val_one, h]
-  · exact h
+  refine hzero.resolve_left fun h => ha ?_
```

Comment:
Use "\maps" instead of "=>" while you're at it?

Reply:
(Not official or anything, but my preference is against "while you're at it" suggestions. They really do, in my opinion, add friction to getting stuff done.)

Reply:
I understand this point!
My point was this is a *new* proof written - so reviewing doesn't become any harder this way. *Authoring* it does; thanks for pointing this out.

Reply:
Let me add: I can make sure to, in the future,
- make such comments directly as suggestions (minimizing the overhead for authors and reviewers)
- limit these comments to new proofs, or PRs cleaning up miscellaneous things, where such a change is in scope.